diff --git a/runtime/Cpp/runtime/src/atn/ATNDeserializationOptions.cpp b/runtime/Cpp/runtime/src/atn/ATNDeserializationOptions.cpp
index a406c4e13..73397107d 100755
--- a/runtime/Cpp/runtime/src/atn/ATNDeserializationOptions.cpp
+++ b/runtime/Cpp/runtime/src/atn/ATNDeserializationOptions.cpp
@@ -7,8 +7,6 @@
 
 using namespace antlr4::atn;
 
-ATNDeserializationOptions ATNDeserializationOptions::defaultOptions;
-
 ATNDeserializationOptions::ATNDeserializationOptions() {
   InitializeInstanceFields();
 }
@@ -22,6 +20,7 @@ ATNDeserializationOptions::~ATNDeserializationOptions() {
 }
 
 const ATNDeserializationOptions& ATNDeserializationOptions::getDefaultOptions() {
+  static ATNDeserializationOptions defaultOptions;
   return defaultOptions;
 }
 
